share/github-backup-utils/ghe-restore-repositories

```diff
@@ -140,20 +143,37 @@ fi
 # rsync all the repository networks to the git server where they belong.
 # One rsync invocation per server available.
 bm_start "$(basename $0) - Restoring repository networks"
+rsync_commands=()
 for file_list in $tempdir/*.rsync; do
   if $CLUSTER; then
     server=$(basename $file_list .rsync)
   else
     server=$host
   fi
-  ghe_verbose "* Transferring repository networks to $server ..."
-  ghe-rsync -avrR --delete \
-    -e "ssh -q $opts -p $port $ssh_config_file_opt -l $user" \
-    --rsync-path="sudo -u git rsync" \
-    --files-from=$file_list \
-    "$GHE_DATA_DIR/$GHE_RESTORE_SNAPSHOT/repositories/./" \
-    "$server:$GHE_REMOTE_DATA_USER_DIR/repositories/" 1>&3
+
+  rsync_commands+=("
+    if [ -n \"$GHE_VERBOSE\" ]; then
```

Comment:
Do we want to do this some other way? Since `ghe_verbose` isn't accessible while running with `parallel` we currently have to do it this way, but it seems a little clunky.

Reply:
I'm pretty certain that outputting to fd3 does work in parallel. You can safely remove this if statement.

Reply:
Outputting to fd3 does work, however `ghe_verbose` guards on `$GHE_VERBOSE` being set prior to outputting: https://github.com/github/backup-utils/blob/9449255689b7b05a6af685891ef3fa206d92c913/share/github-backup-utils/ghe-backup-config#L325-L329

The intention was to mimic `ghe_verbose`, but I guess we can remove the if statement.

`ghe_verbose` doesn't work because we're not running these commands within the environment of this script with `parallel`, we'd need to `source ghe-backup-config` in the generated `rsync_commands` to provide access to it.

Reply:
Oh I see your intention. Thanks for explaining. Let's leave it as is.